features/finder_frontend.feature

```diff
@@ -19,16 +19,6 @@ Feature: Finder Frontend
     And I should see an input field to search
     And I should see an open facet titled "Organisation" with non-blank values
 
-  @normal
-  Scenario: check policy alpha loads
-    When I visit "/government/policies/all"
-    Then I should see "All policy content"
-    And I should see an input field to search
-    And I should see an open facet titled "Document type" with non-blank values
-    And I should see a closed facet titled "Organisation" with non-blank values
-    And I should see a closed facet titled "Policy" with non-blank values
-    And I should see a closed facet titled "People" with non-blank values
```

Comment:
I think these three lines should be added to another test. They were added in commit b40f5d01b9ea900e255a2a56aa7cde3638c930f9 to highlight when things (people etc) are missing from rummager.

cc @MatMoore